setup.cfg

```diff
@@ -1,36 +1,46 @@
+[metadata]
+name = notion-client
+version = 0.9.0
+url = https://github.com/ramnes/notion-sdk-py
+author = Guillaume Gelin
+author_email = contact@ramnes.eu
+description = Python client for the official Notion API
+long_description = file: README.md
+long_description_content_type = text/markdown
+python_requires = >=3.7, <4
+install_requires = httpx >= 0.15.0
+classifiers=
+	Programming Language :: Python
+	Programming Language :: Python :: 3
+	Programming Language :: Python :: 3.7
+	Programming Language :: Python :: 3.8
+	Programming Language :: Python :: 3.9
+	Programming Language :: Python :: 3.10
+	Development Status :: 4 - Beta
+	License :: OSI Approved :: MIT License
```

Comment:
Please stay consistent by using spaces instead of tabs, and ideally please add a TOML linter in `.pre-commit-config.yaml`. :)